deno/fhirchat-demo/build-css.ts

```diff
@@ -67,10 +67,12 @@ async function buildStyles(path: string) {
     });
 
     const __dirname = resolve();
-    const outputPath = `./static/${relative(
-      __dirname,
-      path.replace(".scss", ".css").replace(".kiss", ".css")
-    )}`;
+    const outputPath = `./static/${
+      relative(
+        __dirname,
+        path.replace(".scss", ".css").replace(".kiss", ".css"),
```

Comment:
Since we removed the `kiss` grids 😆  we should remove the file name handling in here too.